Sources/COWMacros/SyntaxExtensions.swift

```diff
@@ -382,6 +495,37 @@ extension InitializerDeclSyntax {
     return signatureStandin == other.signatureStandin
   }
   
+  // Decodable:
+  // init(from decoder: any Decoder) throws
+  var likelyToConformToDecodable: Bool {
```

Comment:
Explicit `internal` access control is recommended. This convention is learned from the Swift Standard Library.